.travis.yml

```diff
@@ -9,13 +9,13 @@ script:
 - echo "Running Unit Tests"
 - docker run -e board_id -e list_id_Done -e list_id_Pending -e list_id_ToDo -e MONGO_LOGIN -e MONGO_PASS todo-app:test Test/view_model_test.py
 
-# Docker Run to Execute Unit Integration Package
+# Docker Run to Execute Unit Integration Package - failed
 - echo "Running Integration Tests"
 - docker run -e board_id -e list_id_Done -e list_id_Pending -e list_id_ToDo -e MONGO_LOGIN -e MONGO_PASS todo-app:test Test/client_test.py
 
 # Docker Run to Execute Unit E2E Selenium Package
-# - echo "Running E2E Tests"
-- docker run -e board_id -e list_id_Done -e list_id_Pending -e list_id_ToDo -e MONGO_LOGIN -e MONGO_PASS todo-app:test Test/selenium_test.py
+- echo "Running E2E Tests"
+- docker run -e board_id -e list_id_Done -e list_id_Pending -e list_id_ToDo -e MONGO_LOGIN -e MONGO_PASS -e ANON_USER -e ANON -e SECRET_KEY todo-app:test Test/selenium_test.py
```

Comment:
These arguments could be just specified in our test setup code (like the `Login_Disabled` one is) - without that step your readme instructions for running the tests fail since the user doesn't know they need to set up those variables.

Reply:
Extra note - I don't think your app ever looks for an `ANON` environment variable?